tools/encyclopedia/macros.xml

```diff
@@ -0,0 +1,509 @@
+                <param argument="-minLength" type="integer" value="5" min="1" max="100" label="minLength" optional="true"/>
+                <param argument="-maxLength" type="integer" value="100" min="10" max="200" label="maxLength" optional="true"/>
+                <param argument="-minEluteTime" type="integer" value="12" min="1" max="69" label="minEluteTime" optional="true"/>
+                <param argument="-maxMissedCleavage" type="integer" value="1" min="0" max="5" label="maxMissedCleavage" optional="true"/>
+                <param argument="-minQuantitativeIonNumber" type="integer" value="3" min="1" max="10" label="minQuantitativeIonNumber" optional="true"/>
+                <param argument="-minNumOfQuantitativePeaks" type="integer" value="3" min="1" max="10" label="minNumOfQuantitativePeaks" optional="true"/>
+                <param argument="-numberOfQuantitativePeaks" type="integer" value="3" min="1" max="10" label="numberOfQuantitativePeaks" optional="true"/>
+<!--
+                <param argument="-addDecoysToBackground" type="boolean" truevalue="true" falsevalue="false" checked="false" label="addDecoysToBackground"/>
+                <param argument="-dontRunDecoys" type="boolean" truevalue="true" falsevalue="false" checked="false" label="dontRunDecoys"/>
+-->
+            </when>
+        </conditional>
+    </xml>
+    <token name="@SEARCH_OPTIONS@">
+    #if $options.search.set_search == 'yes'
+        -minCharge $options.search.minCharge
+        -maxCharge $options.search.maxCharge
+        -minLength $options.search.minLength
+        -maxLength $options.search.maxLength
+        -minEluteTime $options.search.minEluteTime
+        -maxMissedCleavage $options.search.maxMissedCleavage
+        -minQuantitativeIonNumber $options.search.minQuantitativeIonNumber
+        -minNumOfQuantitativePeaks $options.search.minNumOfQuantitativePeaks
+        -numberOfQuantitativePeaks $options.search.numberOfQuantitativePeaks
+        ## -addDecoysToBackground $options.search.addDecoysToBackground
+        ## -dontRunDecoys $options.search.dontRunDecoys
+    #end if
+    </token>
+
+    <xml name="options_section">
+        <section name="options" title="Parameter Settings" expanded="false">
+            <expand macro="common_options"/>
+            <expand macro="mass_library_tolerance"/>
+            <expand macro="percolator_options"/>
+            <expand macro="peak_options"/>
+            <expand macro="window_options"/>
+            <expand macro="modification_options"/>
+            <expand macro="search_options"/>
+        </section>
+    </xml>
+
+    <xml name="libexport">
+        <param argument="-a" type="boolean" truevalue="true" falsevalue="false" checked="false" label="align between files"/>
+    </xml>
+    <token name="@MSCONVERT_CMD@"><![CDATA[
+      msconvert  --zlib --64 --mzML --simAsSpectra --filter "peakPicking true 1-" --filter "demultiplex optimization=overlap_only" *.raw
```

Comment:
You have not defined msconvert as a dependency, isn't it? I would not include msconvert into this tool. msconvert only runs in a Docker container ... so we would restrict this tool to Docker only.

Reply:
This is just part of the HELP text for the tools.   
The protocol for both the Mass Spec and the conversion of the Mass Spec raw output to mxML format is critical for getting good results from EncyclopeDIA.